launcher/resources/git-shim.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+import socket, os, os.path, sys
+
+PORT=20280
+BUF_SIZE=4096
+
+def receive_all(sock):
+    """Receive all data coming through the socket and return it as bytes."""
```

Comment:
Is this comment accurate? It seems like it's returning the data as a decoded UTF string, rather than as raw bytes